doc/changes/latest.inc

```diff
@@ -103,6 +105,8 @@ Enhancements
 Bugs
 ~~~~
 
+- :func:`mne.time_frequency.tfr_array_multitaper` now returns results per taper when ``output='complex'`` (:gh:`10281` by `Mikołaj Magnuski`)
```

Comment:
Yay RST!
```suggestion
- :func:`mne.time_frequency.tfr_array_multitaper` now returns results per taper when ``output='complex'`` (:gh:`10281` by `Mikołaj Magnuski`_)
```
But this also needs to go below any `:newcontrib:` entries (those should be at the top)